src/ctim/schemas/vocabularies.cljc

```diff
@@ -427,7 +427,9 @@
     "sighting-of"
     "targets"
     "uses"
-    "variant-of"})
+    "variant-of"
+    "technique-of"
```

Comment:
What is the purpose of the `technique-of` relation type? Between what type of entities would it be used?

Could you also update the documentation https://github.com/threatgrid/ctim/blob/master/doc/defined_relationships.md that defines how to use relation types?